project_scrum/project_scrum.py

```diff
@@ -491,7 +491,9 @@ class ProjectScrumTest(models.Model):
     user_story_id_test = fields.Many2one(comodel_name="project.scrum.us", string="User Story")
     description_test = fields.Html(string='Description')
     sequence_test = fields.Integer(string='Sequence', select=True)
-    stats_test = fields.Selection([('draft', 'Draft'), ('in progress', 'In Progress'), ('cancel', 'Cancelled')], string='State', required=False)
+    stats_test = fields.Selection([('draft', 'Draft'),
+                                   ('in progress', 'In Progress'),
+                                   ('cancel', 'Cancelled')], string='State', required=False)
```

Comment:
To be PEP8 compliant, the line break must be right after the `(`, so that the `cr, uid,`  go at the beginning of the next line.
Anyway, teh `cr, uid,`  should go away soon, when the port to the new API.